src/api/server/common.rst

```diff
@@ -1354,150 +1358,139 @@ error.
-+----------------+------------------------------------------------------+----------+
-| ``405``        | Number of HTTP 405 Method Not Allowed responses      | number   |
-+----------------+------------------------------------------------------+----------+
-| ``409``        | Number of HTTP 409 Conflict responses                | number   |
-+----------------+------------------------------------------------------+----------+
-| ``412``        | Number of HTTP 412 Precondition Failed responses     | number   |
-+----------------+------------------------------------------------------+----------+
-| ``500``        | Number of HTTP 500 Internal Server Error responses   | number   |
-+----------------+------------------------------------------------------+----------+
-
-``httpd``
-=========
-
-.. lint: ignore errors for the next 13 lines
-
-+----------------------------------+----------------------------------------------+----------+
-| Statistic ID                     | Description                                  | Unit     |
-+==================================+==============================================+==========+
-| ``bulk_requests``                | Number of bulk requests                      | number   |
-+----------------------------------+----------------------------------------------+----------+
-| ``clients_requesting_changes``   | Number of clients for continuous _changes    | number   |
-+----------------------------------+----------------------------------------------+----------+
-| ``requests``                     | Number of HTTP requests                      | number   |
-+----------------------------------+----------------------------------------------+----------+
-| ``temporary_view_reads``         | Number of temporary view reads               | number   |
-+----------------------------------+----------------------------------------------+----------+
-| ``view_reads``                   | Number of view reads                         | number   |
-+----------------------------------+----------------------------------------------+----------+
+- ``counter``: Monotonically increasing counter, resets on restart
+- ``histogram``: Binned set of values with meaningful subdivisions
+- ``gauge``: Single numerical value that can go up and down
 
 You can also access individual statistics by quoting the statistics sections
 and statistic ID as part of the URL path. For example, to get the
-``request_time`` statistics, you can use:
+``request_time`` statistics within the ``couchdb`` section for the target
+node, you can use:
 
 .. code-block:: http
 
-    GET /_stats/couchdb/request_time HTTP/1.1
+    GET /_node/_local/_stats/couchdb/request_time HTTP/1.1
 
 This returns an entire statistics object, as with the full request, but
-containing only the request individual statistic. Hence, the returned structure
-is as follows:
+containing only the request individual statistic.
```

Comment:
I know you didn't add this wording, but "request individual statistic" is a bit confusing to me. Is there a way to clarify that?  I'm not even sure what it means, so I don't have a suggestion to offer myself.